packages/website/src/components/PlaygroundPage/index.tsx

```diff
@@ -79,34 +80,54 @@ export const PlaygroundPage: FC<{ version: string | null }> = (props) => {
 
   const { height, ref } = useAvailableHeight();
 
+  /**
+   * I couldn't call `use()` here, probably because Nextra doesn't support `app/` yet, and Next.js with `pages/` uses an older React version.
+   * Next.js fails with this error: "TypeError: (0 , react__WEBPACK_IMPORTED_MODULE_3__.use) is not a function"
+   * Waiting for React 19 to clean this up...
+   */
```

Comment:
Notice the comment. React haven't had new releases since June 2022, but I heard on Twitter React 19 might happen relatively soon.